testsuite/MDAnalysisTests/analysis/test_wbridge.py

```diff
@@ -197,7 +197,7 @@ def test_count_by_type_multiple_frame(self):
         wb = WaterBridgeAnalysis(u, 'protein and (resid 1)', 'protein and (resid 4)')
         # Build an dummy WaterBridgeAnalysis object for testing
         wb._timeseries = True
-        wb.timesteps = range(6)
+        wb.timesteps = [0, 1, 2, 3, 4, 5]
```

Comment:
do you still know the error that was thrown? I don't get any errors on test with the newest pylint and the develop branch.

The code itself is equivalent.